saml2/saml2-service-provider/src/test/java/org/springframework/security/saml2/provider/service/authentication/OpenSaml4AuthenticationProviderTests.java

```diff
@@ -51,16 +51,20 @@
 import org.opensaml.saml.saml2.core.EncryptedAssertion;
 import org.opensaml.saml.saml2.core.EncryptedAttribute;
 import org.opensaml.saml.saml2.core.EncryptedID;
+import org.opensaml.saml.saml2.core.Issuer;
```

Comment:
On each file that you modify, will you please change any copyright message of the form `20xx-20yy` copyright date to `20xx-2024`?

Reply:
Fixed! Thanks!